### DISCUSSION TITLE
[MATCH] Issue with provisioning profile

### DISCUSSION THREAD

**Discussion by tuanndwork** [fastlane/fastlane#17373] [Q&A] [answered]
Anyone here have issue with match and build fail as image below:
I run match ad-hoc for iOS app but when run build have failed with status 65.

error: Provisioning profile "match AdHoc ..." doesn't include signing certificate "iPhone Distribution: ...". (in target 'ReleaseApp' from project '...')

After that, I open xcode and show other message 
![adhoc-1](https://user-images.githubusercontent.com/67319472/95053568-d04a3380-071a-11eb-805a-1c5eed1e8804.png)

And absolute when I go to developer account manage and edit provisioning profile use app distribution certificate will be ok.

**Comment by rogerluan** [comment, marked as answer]
I've faced similar issues like this in the past (a couple times, actually). The project I work on has "iCloud" capability, and whenever this happens (happens every time we register new Device IDs), we have to turn iCloud off, build (fails), and then turn iCloud ON again, build (succeeds), and discard all changes generated in git. This is a bug we've been seeing since Xcode 10 (IIRC - maybe 9) 😞 

Let me know if that works for you, as it may be something unrelated to the bug that I see. 💪

**Reply by tuanndwork** [reply]
Thank you...  I've found fastlane's issue before. and found this https://github.com/fastlane/fastlane/issues/15031
I just use new pipeline with flag "generate_apple_certs" is true

**Reply by rogerluan** [reply]
Oh, I'm glad you sorted it out! 🙌

**Reply by rogerluan** [reply]
You might want to mark this reply of yours as being `the answer` so people can easily find it in the future 😊
